recipes/plannotate/meta.yaml

```diff
@@ -1,48 +1,46 @@
-{% set version = "1.2.0" %}
+{% set version = "1.2.1" %}
 
 package:
   name: plannotate
   version: {{ version }}
 
 build:
-  number: 4
+  number: 1
   noarch: python
 
 source:
   - url: https://github.com/barricklab/pLannotate/archive/refs/tags/v{{version}}.tar.gz
     sha256: d58d11592d62efdcf6834c6aa6188f58efb1129132e13c6342f1a523e0aa7497
-    patches:
-      - patch
   - url: https://github.com/barricklab/pLannotate/releases/download/v{{version}}/BLAST_dbs.tar.gz
     sha256: 34c7bacb1c73bd75129e16990653f73b3eba7e3cdb3816a55d3989a7601f2137
     folder: BLAST_dbs
 
 requirements:
   host:
-    - python >=3.7
+    - python>=3.9
     - pip
   run:
-    - altair =4.2.2
-    - biopython =1.78
-    - bokeh =2.4.1
-    - click =7.1.2
+    - click
     - curl
-    - blast =2.10.1
-    - diamond =2.0.13
-    - numpy 1.21.5
-    - pandas 1.3.5
-    - python =3.7
-    - ripgrep =13.0.0
-    - streamlit =1.8.1
-    - tabulate =0.8.9
-    - trnascan-se =2.0.7
+    - numpy
+    - biopython>=1.78
+    - blast>=2.10.1
+    - diamond>=2.0.13
+    - pandas>=1.3.5
+    - python>=3.9
+    - ripgrep>=13.0.0
+    - tabulate>=0.8.9
+    - trnascan-se>=2.0.7
+    - streamlit=1.8.1
+    - altair=4.2.*
+    - bokeh=2.4.1
 
 test:
   commands:
     - plannotate batch --help
 
 about:
-  home: https://github.com/barricklab/pLannotate
+  home: https://github.com/mmcguffi/pLannotate
   license: GPL-3
```

Comment:
can you please use here SPDX identifiers?

Reply:
Im not sure I understand: 
```
license: GPL-3
license_file: LICENSE
```
These are already in the meta.yam?